pkg/api/v1/pod/util.go

```diff
@@ -52,7 +52,7 @@ func FindPort(pod *v1.Pod, svcPort *v1.ServicePort) (int, error) {
 type ContainerType int
 
 const (
-	// Containers is for normal containers
+	// Containers are for normal containers
```

Comment:
nit: I think it's a bit confusing, but I believe "is" is correct here, since constant is still singular, just like `InitContainers` and `AllContainers`.

Reply:
Will resolve